core/src/dbs/notification.rs

```diff
@@ -21,6 +23,7 @@ impl Display for Action {
 }
 
 #[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+#[revisioned(revision = 1)]
```

Comment:
I understand that this versions the structs but is there some documentation on this? It's not a challenge for this PR it's just that this PR focuses a lot on this mechanism and I'd like to understand it more

Reply:
Hey Maxwell, it's using the `revision` crate. You can find some docs in the [README](https://github.com/surrealdb/revision) and in the [Rust docs](https://docs.rs/revision/latest/revision/attr.revisioned.html).